src/web/mjs/connectors/ReaperScans.mjs

```diff
@@ -11,8 +10,142 @@ export default class ReaperScans extends WordPressMadara {
+                referrer: requestdata.referrer,
+                referrerPolicy: 'strict-origin-when-cross-origin',
+                body: JSON.stringify(requestdata.payload),
+                credentials: 'include',
+                headers: {
+                    'Accept': 'text/html, application/xhtml+xml',
+                    "Content-Type": "application/json",
+                    "X-CSFRF-TOKEN": requestdata.token,
+                    "X-Livewire": "true",
+                    'x-referrer': requestdata.referrer,
+                }
+            });
+            response = await fetch(request);
+            let data = await response.json();
+            //data contains the html nodes with the chapters
+            //and the informations needed to request the next page
+            doc = parser.parseFromString(data.effects.html, 'text/html');
+            chapters = this.getChaptersFromDoc(doc);
+            chapters.length > 0 ? chapterList.push(...chapters) : run = false;
+            //update payload data using response data, for next fetch
+            requestdata.payload.serverMemo.checksum = data.serverMemo.checksum;
+            requestdata.payload.serverMemo.htmlHash = data.serverMemo.htmlHash;
+            requestdata.referrer = new URL(manga.id+"?page="+page, this.url).href;
+            requestdata.payload.serverMemo.data.page = page;
+            requestdata.payload.serverMemo.data.paginators.page = page;
+            requestdata.payload.updates[0].payload.params[0] = page+1;
+            await this.wait(this.config.throttle.value);
+        }
+        return chapterList;
+    }
+    getChaptersFromDoc(doc) {
+        let chapterList = [];
+        //get chapters
+        let data = doc.querySelectorAll(this.queryChapters);
+        data.forEach(element => {
+            let chapter = {
+                id: element.pathname,
+                title : element.querySelector('p').textContent.trim()
+            };
+            chapterList.push(chapter);
+        });
+        return chapterList;
+    }
+    createPayloadForChapterPages(doc) {
+        const token = doc.querySelector('meta[name="csrf-token"]').getAttribute("content");
+        const payloadid = (Math.random() + 1).toString(36).substring(8);
+        let wiredata = JSON.parse(doc.querySelector('.max-w-6xl div[wire\\:initial-data]').getAttribute("wire:initial-data"));
+        const referrer = wiredata.effects.path;//of course the current url could be used too and i
+        delete wiredata.effects;
+        wiredata.serverMemo.data.page = 1;
+        wiredata.serverMemo.data.paginators.page = 1;
+        wiredata['updates']=[{
+            type: "callMethod",
+            payload: {
+                id: payloadid,
+                method: "gotoPage",
+                params: [
+                    2,
+                    "page"
+                ]
+            }
+        }
+        ];
+        return {
+            payload : wiredata, token :token, referrer : referrer};
```

Comment:
lint, missing new indented line before `}`

Reply:
Lint processor (from the PR) says there is no pb.  Copied eslint rules from the repo, and no pb on my end too.
I dont get it

Reply:
the closing bracket is not at the same level as the return, that's a code formating issue (lint), but it looks like it doesn't trigge rthe lint validator

Reply:
```
        return {
            payload : wiredata, token :token, referrer : referrer};
```
to
```
        return {
            payload : wiredata, token :token, referrer : referrer
        };
```